CouchBaseStorage/Configuration/OrleansDocumentExpiry/OrleansGrainExpiryConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace CouchBaseProviders.Configuration.OrleansDocumentExpiry
+{
+    public static class OrleansGrainExpiryConstants
+    {
+        /// <summary>
+        /// Path to target <see cref="OrleansConfigurationSection"/> configuration XML element.
+        /// </summary>
+        public static readonly string OrleansConfigurationSectionPath = "orleans";
+
+        public const string GrainExpiryCollectionName = "grainExpiry";
+
+        public const string GrainTypePropertyName = "grainType";
+
+        public const string ExpiryPropertyName = "timespan";
```

Comment:
I think the property name of `expiresAt` or `expiresIn` is more accurate than `timespan`.

Reply:
I originally named it expiry but I thought that timespan made it more clear the type of data expected. I've renamed it to expiresIn now.

The readme file should probably have a section relating to expiry configuration to make this clear to consumers of this provider.